frontend/src/utils/constants.js

```diff
@@ -93,6 +99,8 @@ export const APPLICATION_ERROR_MESSAGES = Object.freeze({
   FACILITY_TYPE: 'Facility type required',
   OPERATIONAL_COST: 'Operating costs required -or- Facility costs required',
   STAFFING: 'Staffing information required',
+  TRANSPORT: 'Transportation information required',
+  OTHER: 'Description of other costs required',
```

Comment:
could we rename these constants to SUPP_TRANSPORT and SUPP_OTHER? so we will know these constants are for SUPP app

Reply:
sure :)